icu4c/source/test/intltest/dtptngts.cpp

```diff
@@ -1393,7 +1393,7 @@ void IntlTestDateTimePatternGeneratorAPI::testJjMapping() {
         // Now check that shortPattern and jPattern use the same hour cycle
         if ((uprv_strncmp(localeID, "csw", 3) == 0 || uprv_strncmp(localeID, "kxv_", 4) == 0 || uprv_strncmp(localeID, "zh_Hans_MY", 10) == 0
                 || uprv_strncmp(localeID, "yue_Hant_CN", 11) == 0) 
-        		&& logKnownIssue("CLDR-17199", "Need timeFormats with h for csw, kxv_Xxxx, zh_Hans_MY, yue_Hant_CN")) {
+        		&& logKnownIssue("CLDR-17979", "Need timeFormats with h for csw, kxv_Xxxx, zh_Hans_MY, yue_Hant_CN")) {
```

Comment:
Is it still failing for all four? csw, kxv_*, zh_Hans_MY, yue_Hant_CN?

Reply:
Revised this, removing passing locales. Now it's only yue_Hant_CN that gives a test error.